pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dao/kit/KitDao.java

```diff
@@ -450,6 +450,7 @@ private int deleteKitRequest(Connection conn, int kitRequestId) throws SQLExcept
         }
     }
 
+    @VisibleForTesting
```

Comment:
I don't think we need this annotation on a public method on a dao.  It's general purpose.

Reply:
Noted, will remove it!